Makefile

```diff
@@ -45,5 +45,8 @@ clean:
 	rm -f policy.wasm annotated-policy.wasm
 
 .PHONY: e2e-tests
-e2e-tests: annotated-policy.wasm
+e2e-tests: policy.wasm
+	# The policy annonation uses a different metadata file for testing because the 
+	# e2e tests require access to ConfigMap to test the parameters feature.
+	kwctl annotate -m test_data/metadata.yml -u README.md -o annotated-policy.wasm policy.wasm
```

Comment:
I wonder if we should just have the contextAwareResources stanza in a `test_data/context_aware_res.yaml` and just concatenate it to the original metadata.yml when we are running the test. It's more DRY.